ext/ldap/config.m4

```diff
@@ -79,7 +79,8 @@ if test "$PHP_LDAP" != "no"; then
     LDAP_PTHREAD=
   fi
 
-  MACHINE_INCLUDES=$($CC -dumpmachine)
+  dnl -pc removal is a hack for clang
+  MACHINE_INCLUDES=$($CC -dumpmachine | sed 's/-pc//')
```

Comment:
```suggestion
  MACHINE_INCLUDES=$($CC -dumpmachine | $SED 's/-pc//')
```

And also maybe if the `sed` is used here there is also a `$SED` variable created in the Makefile.